dashboard/app/views/sections/show.html.haml

```diff
@@ -78,7 +78,7 @@
     - if @section.login_type == Section::LOGIN_TYPE_WORD
       %div#secret{style: 'display: none;'}
         %h4.instructions= t('signinsection.words')
-        = text_field_tag :secret_words
+        = text_field_tag :secret_words, nil, autocomplete: 'off'
```

Comment:
here's the doc for text_field_tag: http://apidock.com/rails/ActionView/Helpers/FormTagHelper/text_field_tag

The 2nd argument is the value (which sets the value attribute for the input tag). It defaults to nil. If we want to specify the 3rd argument (which is where additional attributes like autocomplete go), we have to include the nil 2nd argument.